php/terminus.php

```diff
@@ -18,17 +20,25 @@
 
 \Terminus\Utils\load_dependencies();
 
-# Set a custom exception handler
-set_exception_handler('\Terminus\Utils\handle_exception');
+set_exception_handler('\Terminus\Utils\handle_exception'); # Set a custom exception handler
 
-if (isset($_SERVER['TERMINUS_HOST']) && $_SERVER['TERMINUS_HOST'] != '') {
-  define( 'TERMINUS_HOST', $_SERVER['TERMINUS_HOST'] );
-  \cli\line(\cli\Colors::colorize('%YNote: using custom target "'. $_SERVER['TERMINUS_HOST'] .'"%n'));
-}
-else {
-  define( 'TERMINUS_HOST', 'dashboard.getpantheon.com' );
-}
-define( 'TERMINUS_PORT', '443' );
+if(isset($_SERVER['TERMINUS_HOST']) && $_SERVER['TERMINUS_HOST'] != '') 
+  define('TERMINUS_HOST', $_SERVER['TERMINUS_HOST']);
+else
+  define('TERMINUS_HOST', 'dashboard.getpantheon.com');
 
+define('TERMINUS_PORT', '443');
+
+if(isset($_SERVER['VCR_CASSETTE'])) {
+  \VCR\VCR::configure()->enableRequestMatchers(array('method', 'url', 'body'));
+  \VCR\VCR::configure()->setMode('once');
```

Comment:
Are you having to change this when developing? Can we make it so that if it's in `CI` it runs as `once`, otherwise in development it runs in the `record_new` mode?